packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx

```diff
@@ -20,6 +20,7 @@ type VisualizationContext = {
     chartRef: RefObject<EChartsReact>;
     chartType: ChartType;
     cartesianConfig: ReturnType<typeof useCartesianChartConfig>;
+    bigNumberConfig: ReturnType<typeof useBigNumberConfig>;
```

Comment:
We can remove the individual props (`bigNumber` `bigNumberLabel` etc,) from the context with this change 🙏

Reply:
That's what I did already, right ? Did I miss anything ? https://github.com/lightdash/lightdash/pull/1999/files#diff-2714a1ae3d534b36cee59df77fa420e9008844c61dd318d29e6bcf5d95142166L27 and https://github.com/lightdash/lightdash/pull/1999/files#diff-2714a1ae3d534b36cee59df77fa420e9008844c61dd318d29e6bcf5d95142166L151